config/locales/en.yml

```diff
@@ -1133,10 +1133,17 @@ en:
     logged_out: "Successfully logged out."
   notes_controller:
     research_notes: "Research notes"
-    thank_you_for_contribution: "Success! Thank you for contributing open research,
-      and thanks for your patience while your post is approved by <a href='/wiki/moderation'>community
-      moderators</a> and we'll email you when it is published. In the meantime, if
-      you have more to contribute, feel free to do so."
+   thank_you_for_contribution:  |
```

Comment:
I also think that there may need to be only one space before the `|` character here.

Reply:
should i replace the lines in red with the lines in green or should i just fix the spacing